lib/src/components/Tabs/utils.tsx

```diff
@@ -0,0 +1,17 @@
+import { Icon } from '@/components/Icon'
+
+import type { TabProps } from './types'
+
+export function getIcon({
+  icon,
+  iconColor,
+  isActive,
+}: Pick<TabProps, 'icon' | 'iconColor'> & { isActive: boolean }) {
+  const iconClassName = `text-background-accent-${iconColor}-${isActive ? 'strong' : 'primary'}`
```

Comment:
This is not how you are supposed to build dynamic classes with Tailwind:
https://tailwindcss.com/docs/detecting-classes-in-source-files

<img width="717" height="606" alt="Capture d’écran 2025-11-13 à 11 26 35" src="https://github.com/user-attachments/assets/33ac94e0-4d05-4f92-91a2-ff2ee736787f" />